### PR TITLE
Improve logger

### PR DESCRIPTION
**Pull request by chenhunghan**
Closes https://github.com/lensapp/lens-platform-sdk/issues/60

### CODE REVIEW

**Comment by panuhorsmalahti** [comment]
Can you update README to document the new option?

**Comment by chenhunghan** [comment]
> Can you update README to document the new option?

Added https://github.com/lensapp/lens-platform-sdk/pull/94/commits/25a0f0b6eb4ef381a108abb3cb2b168915055cec

File: src/LensPlatformClient.ts [hunk 1]
```diff
@@ -6,9 +6,8 @@ import { PermissionsService } from "./PermissionsService";
 import { InvitationService } from "./InvitationService";
 import { PlanService } from "./PlanService";
 import axios, { AxiosRequestConfig } from "axios";
-
+import pino from "pino";
 import decode from "jwt-decode";
-import _console from "./helpers/_console";
```
Comment (nevalla) [inline src/LensPlatformClient.ts:11]: do we still use `_console` somewhere?

Comment (chenhunghan) [reply src/LensPlatformClient.ts:11]: in ext there was a legacy version, but it's a different instance from this.

**Review by panuhorsmalahti — APPROVED** [review]

**Review by nevalla — APPROVED** [review]
LGTM

**Pull request merged**